### PR TITLE
Standardize use of range operator (..)

### PR DESCRIPTION
**Pull request by eksperimental**
...by removing spaces around it

### CODE REVIEW

File: lib/elixir/test/elixir/stream_test.exs [hunk 1]
```diff
@@ -337,7 +337,7 @@ defmodule StreamTest do
   test "flat_map/2 with inner flat_map/2" do
     stream = Stream.flat_map(1..5, fn x ->
       Stream.flat_map([x], fn x ->
-        x .. x * x
+        x..x * x
```
Comment (ericmj) [inline lib/elixir/test/elixir/stream_test.exs]: Add parenthesis here to make the precedence clear.

**Comment by josevalim** [comment]
What was the reasoning for settling on no spaces instead of using spaces? Adding spaces would at least be consistent, no?

**Comment by ericmj** [comment]
No spaces looks better to me. Both would be consistent I think.

On Sunday, 21 February 2016, José Valim <notifications@github.com> wrote:

> What was the reasoning for settling on no spaces instead of using spaces?
> Adding spaces would at least be consistent, no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/4312#issuecomment-186797941>.
>


-- 
Eric Meadows-Jönsson

**Comment by lexmag** [comment]
> What was the reasoning for settling on no spaces instead of using spaces?

Besides "no spaces" looks better to me as well, that's the style we use for inspect.

**Pull request merged**